diameter_synthesis/build_diameters.py

```diff
@@ -315,7 +315,7 @@ def _diametrize_neuron(params_tree, neuron, params, neurite_types, config, rng=n
     if params_tree["with_asymmetry"]:
         # Get sections on the major branch
         for apical_section in params.get("apical_point_sec_ids", []):
-            for sec in neuron.sections[apical_section].iter(IterType.upstream):
+            for sec in neuron.sections[apical_section].parent.iter(IterType.upstream):
```

Comment:
```suggestion
            if neuron.sections[apical_section].parent is None:
                continue
            for sec in neuron.sections[apical_section].parent.iter(IterType.upstream):
```
If the apical section is the first one it will crash, isn't it?

Reply:
ah yes, this is very unlikely, but yes it's safer, thanks!